ui/k-toolbar/main.kytos

```diff
@@ -58,6 +58,11 @@
                     tooltip="Enter the southbound priority"
                     placeholder="southbound priority" icon="arrow-right"></k-input>
 
+            <k-input id="max_paths" v-model:value="max_paths"
+                    title="Max paths to consider|compute"
+                    tooltip="Maximum amount of paths to consider for current_path"
+                    placeholder="Max paths to consider|compute" icon="arrow-right"></k-input>
```

Comment:
```suggestion
                    placeholder="Max paths to consider" icon="arrow-right"></k-input>
```

The `consider|compute` that I suggested was for you to pick one. No need for both.

Reply:
I'll go ahead and commit this since this is a trivial suggestion and then I'll merge the PR too to speed things up.